pkg/reconciler/kubernetes/tektonchain/tektonchain.go

```diff
@@ -469,6 +478,27 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonChain
 		}
 		return nil
 	}
+	// if generatedSigningSecret has been changed for chainsCR, then update the Tekton Chain Secret InstallerSet
+	secretInstallerSetSigningKey, _ := strconv.ParseBool(installedSecretTIS.Annotations[SecretTISSigningAnnotation])
```

Comment:
Does `SecretTISSigningAnnotation` present always in the annotations? on the existing running instance too?

Reply:
No it is added if the secret is generated

Reply:
If the annotation `secretTISSigningAnnotation` not present, return empty string ``, that leads bool parse error,
```
false strconv.ParseBool: parsing "": invalid syntax
```